bundle/pom.xml

```diff
@@ -461,11 +479,16 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.commons.scheduler</artifactId>
-            <version>2.3.4</version>
-            <scope>provided</scope>
+            <groupId>com.adobe.aem</groupId>
+            <artifactId>aem-api</artifactId>
+            <version>6.0.0.1</version>
         </dependency>
+        <dependency>
```

Comment:
Two things here:
1) Did you mean to add this dependency twice?
2) AFAICT, this means we are breaking 5.6.1 compatibility. Is this necessary? My preference would be to cut 1.10.0 with 5.6.1 compatibility and then move forward with 2.0 requiring 6.0 as the minimum. WDYT?